src/Primary_Extension/org/lobobrowser/primary/ext/ComponentSource.java

```diff
@@ -179,7 +179,10 @@ public JMenu getFileMenu() {
     openMenu.add(menuItem("New Window", 'N', KeyStroke.getKeyStroke(KeyEvent.VK_N, CMD_CTRL_KEY_MASK),
         this.actionPool.blankWindowAction));
     openMenu.add(menuItem("Cloned Window", 'C', this.actionPool.clonedWindowAction));
-    openMenu.add(menuItem("File...", 'F', KeyStroke.getKeyStroke(KeyEvent.VK_O, CMD_CTRL_KEY_MASK), this.actionPool.openFileAction));
+    final JMenuItem fileMenuItem = menuItem("File...", 'F', KeyStroke.getKeyStroke(KeyEvent.VK_O, CMD_CTRL_KEY_MASK), this.actionPool.openFileAction);
+    // TODO enable the menu item once access control UI is implemented
+    fileMenuItem.setEnabled(false);
```

Comment:
A flag to `menuItem` would be more "functional", yeah. But if it is used only in this one case, I would leave it aside as a minor thing.